src/ensembl/src/content/app/entity-viewer/shared/components/entity-viewer-sidebar/entity-viewer-sidebar-modal/EntityViewerSidebarModal.tsx

```diff
@@ -41,6 +41,12 @@ const entityViewerSidebarModals: Record<
   )
 };
 
+const entityViewerSidebarModalTitles = {
+  [SidebarModalView.SEARCH]: 'Search',
+  [SidebarModalView.BOOKMARKS]: 'Previously viewed',
+  [SidebarModalView.DOWNLOADS]: 'Downloads'
+};
+
```

Comment:
These titles look quite similar to the ones in the Genome Browser. Can't we put this as a common type in `SidebarModal.tsx'?

Reply:
Andrey and I were discussing this during the code review and we thought that the shared component shouldn't really care about what view/title it is going to display.